### PR TITLE
chore(readme): fixes for typos

### PR DESCRIPTION
**Pull request by bengummer**

### CODE REVIEW

**Comment by mcorbanini** [comment]
LGTM 👍

File: README.md [hunk 1]
```diff
@@ -837,7 +837,7 @@ skate.define('x-tab', {
 });
 ```
 
-It's preferrable not to reach up the DOM hierarchy because that couples your logic to a specific DOM structure that the child has no control over. To decouple this so that your child can be used anywhere, simply trigger an event.
+It's preferable not to reach up the DOM hierarchy because that couples your logic to a specific DOM structure that the child has no control over. To decouple this so that your child can be used anywhere, simply trigger an event.
 
 *Note that events cannot be triggered with `skate.emit()` on disabled elements. Events also can't bubble through disabled elements.*
```
Comment (nadiam84) [inline README.md]: maybe get rid of this since this behavior was changed recently
https://github.com/skatejs/skatejs/pull/727

File: README.md [hunk 2]
```diff
@@ -874,7 +874,7 @@ skate.emit(elem, 'event', {
 
 ### `link (elem, propSpec)`
 
-The `link()` function returns a function that you can bind as an event listener. The handler will take the event and propagte the changes back to the host element. This essentially allows for 2-way data-binding but is safer as the propagation of the user input value back to the component element will trigger a re-render ensuring all dependent UI is up to date.
+The `link()` function returns a function that you can bind as an event listener. The handler will take the event and propagate the changes back to the host element. This essentially allows for 2-way data-binding but is safer as the propagation of the user input value back to the component element will trigger a re-render ensuring all dependent UI is up to date.
```
Comment (nadiam84) [inline README.md]: Feels like this paragraph needs at least two commas...

File: README.md (hunk 1 above)
Comment (bengummer) [inline README.md]: Done :) thanks for noticing

**Comment by nadiam84** [comment]
lgtm

**Pull request merged**